examples/controlnet/train_controlnet_flax.py

```diff
@@ -213,6 +214,17 @@ def parse_args():
         action="store_true",
         help="Load the pretrained model from a PyTorch checkpoint.",
     )
+    parser.add_argument(
+        "--controlnet_revision",
+        type=str,
+        default=None,
+        help="Revision of controlnet model identifier from huggingface.co/models.",
+    )
+    parser.add_argument(
+        "--controlnet_from_pt",
+        action="store_true",
+        help="Load the controlnet model from a PyTorch checkpoint.",
+    )
```

Comment:
@yiyixuxu have we talked about these two arguments in the README?

Reply:
no - should we?  the doc string is not enough?

Reply:
No strong opinion. 

I think it's fine either way.